persistence/provider/sql/eventstore.go

```diff
@@ -64,20 +64,40 @@ type eventStoreDriver interface {
 		ak string,
 	) (uint64, error)
 
-	// SelectEvents selects events from the eventstore that match the given
-	// query.
+	// SelectEventsByType selects events from the eventstore that match the
+	// given event types query.
 	//
 	// f is a filter ID, as returned by InsertEventFilter(). If the query does
 	// not use a filter, f is zero.
-	SelectEvents(
+	SelectEventsByType(
 		ctx context.Context,
 		db *sql.DB,
 		ak string,
 		q eventstore.Query,
 		f int64,
 	) (*sql.Rows, error)
 
-	// ScanEvent scans the next event from a row-set returned by SelectEvents().
+	// SelectEventsBySource selects events from the eventstore that match the
+	// given source, namely the source's key and id. o is an optional offset
+	// parameter from which the messages should be loaded.
```

Comment:
```suggestion
	// SelectEventsBySource selects events from the eventstore that were
	// produced by a specific handler.
```

Trimmed this down, `o` isn't optional.